### PR TITLE
Add fnr (FindAndReplace) v1.8.1

### PR DESCRIPTION
**Pull request by MCOfficer**

### CODE REVIEW

File: fnr.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+    "description": "An open source tool to find and replace text in multiple files.",
+    "homepage": "https://github.com/zzzprojects/findandreplace",
+    "version": "1.8.1",
+    "license": "MIT",
+    "url": "http://findandreplace.io/downloads/fnr.zip",
+    "bin": "fnr.exe",
+    "hash": "205f31941ee62b9b66a5dcbe3c3af5449d54e80095e7304675660951f7fbd4bf",
+    "checkver": {
+        "url": "https://github.com/zzzprojects/findandreplace/releases/",
```
Comment (Ash258) [inline fnr.json]: Change it to `"github": "https://github.com/zzzprojects/findandreplace"`. No need to append releases.

Comment (MCOfficer) [reply fnr.json]: apparently checkver doesn't like that.
```
fnr: couldn't match '/releases/tag/V([\d.]+)' in https://github.com/zzzprojects/findandreplace/
```

Comment (Ash258) [reply fnr.json]: proof

**Comment by Ash258** [comment]
@brandon93s If you make it only github, it's not working. That's why it is this way.

Your version

**Comment by brandon93s** [comment]
@Ash258 It was fixed upstream before making his change: https://github.com/lukesampson/scoop/commit/2e2633e9640f6cab5c2f895b680345cd6ca005c4

**Pull request merged**